package.json

```diff
@@ -1,4 +1,113 @@
-{	
-  "private": true,	
-  "workspaces": ["collect-package-metadata", "service"]	
-}
\ No newline at end of file
+{
+  "name": "package-analyzer",
+  "private": "true",
+  "repository": "git@github.com:Hermanya/package-analyzer.git",
+  "version": "0.1.8",
+  "description": "Package analyzer 🔎 📦",
+  "workspaces": [
+    "packages/*"
+  ],
+  "engines": {
+    "node": ">10.0.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*": [
+      "pretty-quick --staged"
+    ],
+    "*.ts": [
+      "eslint --fix"
+    ]
+  },
+  "prettier": {
```

Comment:
any reason for these prettier changes? I kinda like defaults

Reply:
this is a basic thing we have been using in our all-new projects. no other reason.

Reply:
This basic thing results in changes that make this particular PR quite noisy.
In addition to this, I do prefer defaults, please remove these changes.